django_ses_plus/settings.py

```diff
@@ -3,7 +3,9 @@
 DJANGO_SES_PLUS_SETTINGS = getattr(settings, "DJANGO_SES_PLUS_SETTINGS", {})
 
 DJANGO_SES_PLUS_SETTINGS.setdefault("SEND_EMAIL", True)
-DJANGO_SES_PLUS_SETTINGS["CELERY_TASK_RETRY_KWARGS"].setdefault("CELERY_TASK_RETRY_KWARGS", {'max_retries': 15, 'countdown': 60})
+
+if DJANGO_SES_PLUS_SETTINGS["SEND_EMAIL"]:
+    DJANGO_SES_PLUS_SETTINGS["CELERY_TASK_RETRY_KWARGS"].setdefault("CELERY_TASK_RETRY_KWARGS", {'max_retries': 15, 'countdown': 60})
```

Comment:
These lines should be changed with 
DJANGO_SES_PLUS_SETTINGS.setdefault("CELERY_TASK_RETRY_KWARGS", {'max_retries': 15, 'countdown': 60})

Reply:
I didn't get what you mean here, If I change these lines with the suggested line, then there would be no change. 😅

Reply:
Looks like, it raises KeyError.
`>>> d = {}
>>> d["CELERY_TASK_RETRY_KWARGS"].setdefault("CELERY_TASK_RETRY_KWARGS", {'max_retries': 15, 'countdown': 60})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'CELERY_TASK_RETRY_KWARGS'
>>> d = {}
>>> d.setdefault("CELERY_TASK_RETRY_KWARGS", {'max_retries': 15, 'countdown': 60})
{'max_retries': 15, 'countdown': 60}`

Reply:
Thank you Göksel! Fixed accordingly. I have also ignored `DEFAULT_FROM_EMAIL` setting if `SEND_MAIL` is set to `False`, I guess we won't need it too if `SEND_MAIL` is `False` right?